sip-security/src/test/java/de/ikor/sip/foundation/security/authentication/CompositeAuthenticationFilterTest.java

```diff
@@ -119,18 +114,16 @@ void WHEN_doFilterInternal_WITH_validCredentials_THEN_everythingOk() throws Exce
   @Test
   void WHEN_doFilterInternal_WITH_ignoredRoute_THEN_everythingOk() throws Exception {
     // arrange
-    MockHttpServletRequest request = new MockHttpServletRequest();
     request.setRequestURI("/ignored");
-    MockHttpServletResponse response = new MockHttpServletResponse();
     FilterChain chain = mock(FilterChain.class);
 
     // act
     subject.doFilter(request, response, chain);
+    Authentication contextToken = SecurityContextHolder.getContext().getAuthentication();
```

Comment:
I would leave this line in assert section because its not related to test execution, its there for verification only